Sources/ProjectSpec/Scheme.swift

```diff
@@ -294,8 +304,8 @@ extension Scheme.Test.TestTarget: JSONObjectConvertible {
 
     public init(jsonDictionary: JSONDictionary) throws {
         name = try jsonDictionary.json(atKeyPath: "name")
-        randomExecutionOrder = jsonDictionary.json(atKeyPath: "randomExecutionOrder") ?? false
-        parallelizable = jsonDictionary.json(atKeyPath: "parallelizable") ?? false
+        randomExecutionOrder = jsonDictionary.json(atKeyPath: "randomExecutionOrder") ?? Scheme.Test.TestTarget.randomExecutionOrderDefault
+        parallelizable = jsonDictionary.json(atKeyPath: "parallelizable") ?? Scheme.Test.TestTarget.parallelizableDefault
```

Comment:
Very minor and doesn't need to change, but not sure you need to traverse the whole namespace path, I think you can just use `TestTarget.parallelizableDefault`

Reply:
Thanks!
After trying it out, it seems necessary to specify the complete namespace in the extension.